Dockerfile

```diff
@@ -29,7 +29,12 @@ RUN export DEBIAN_FRONTEND='noninteractive' && \
 	cd .. && \
 	rm -rf dotnet-warmup && \
 	apt-get clean && \
-	rm -rf /var/lib/apt/lists/* /tmp/*
+
+RUN curl -L -o binaries.zip https://www.myget.org/F/starcounter/api/v2/package/runtime.linux-x64.runtime.native.Starcounter.Bluestar2/2.0.2 && \
+	unzip binaries.zip
+
+ENV PATH $PATH:/runtimes/linux-x64/native
+ENV LD_LIBRARY_PATH /runtimes/linux-x64/native
```

Comment:
Since this is a generated image, rather than unzipping in a custom directory and setting `PATH` and `LD_LIBRARY_PATH`, we should unzip and move things to the correct directories.

Executables go into `/usr/local/bin`, shared libraries go into `/usr/local/lib`. Anything else (text files etc) can go into `/opt/starcounter/`.

One approach would be to unzip everything into `/opt/starcounter/bluestar` and then either move the files where they should be or create links to them.

References:
https://www.tldp.org/LDP/abs/html/systemdirs.html
http://tldp.org/HOWTO/Program-Library-HOWTO/shared-libraries.html